eq-author-api/src/validation/schemas/definitions.json

```diff
@@ -4,9 +4,7 @@
     "populatedString": {
       "type": "string",
       "pattern": "\\w+",
-      "errorMessage": {
-        "pattern": "ERR_VALID_REQUIRED"
-      }
+      "errorMessage": "ERR_VALID_REQUIRED"
```

Comment:
In case you're wondering - I made this change so that using the `populatedString` ref will give the error for null values _as well as_ empty strings (this makes the actual AJV schema simpler because there's no need to also check for null). 

Previously it only gave an error when the `pattern` failed a match and ignored errors on `type`. This handles both now.

Once Shane's PR is through some of the cruft could be removed from the AJV schema file since a lot of the checks for null strings AND populatedStrings can be removed after this change.